### ISSUE TITLE
CrashDetect 4.13.1 LSE - Maxing out CPU and RAM.

### ISSUE THREAD

**Issue by Pendulum80** [Zeex/samp-plugin-crashdetect#34]
So I've been having a major issue where for a long period of time my script will keep restarting with full CPU and full RAM maxing out even a full dedicated servers resources and I had no idea why so I did some investigting.

I removed the Gamemode and Filterscripts and it still happened. Upon removal of Crashdetect from the plugin list it started working fine.

I have it running fine from what I can remember on another server so I began to take off the other plugins one by one and it had no effect although but this point the CPU and RAM would go up significantly (4gb RAM 100% CPU) but then drop rapidly on the same PID and the server would run fine. This happened on multiple gamemodes.

Basically their is an insane CPU/RAM spike for using the long script version and I think it gets so high that the process doesn't start up at all and quits alot of the time.

It seems to just about get by loading fine on my 8GB VPS where I have enough RAM for it to nearly exceed the 4GB but, on my smaller VPS it doesn't work at all. I have noticed by having 3 lightweight filterscripts it just doesn't work at all.

I know this is alot of information and is quite confusing but, the tl;dr is.. LSE version makes huge CPU/RAM spikes and servers do not start right as a result. Its a shame because when it works I actually got correct crash reports.

**Comment by Zeex** [comment]
Closed as oudated, feel free to reopen.
